wa/instrumentation/energy_measurement.py

```diff
@@ -284,9 +292,10 @@ def initialize(self, context):
         self.instruments = self.backend.get_instruments(self.target, **self.params)
 
         for instrument in self.instruments.itervalues():
-            if instrument.mode != CONTINUOUS:
+            if not (instrument.mode & CONTINUOUS):
                 msg = '{} instrument does not support continuous measurement collection'
                 raise ConfigError(msg.format(self.instrument))
+            instrument.setup()
```

Comment:
Why does setup get called here?

Reply:
Because we should make sure this happens at least once per run. This is where, e.g., binaries get installed.  See

http://devlib.readthedocs.io/en/latest/overview.html#one-time-setup